chef/cookbooks/utils/metadata.rb

```diff
@@ -0,0 +1,7 @@
+name            "utils"
+maintainer       "Crowbar Project"
+maintainer_email "crowbar@googlegroups.com"
+license          "Apache 2.0"
+description      "Provides utilities for other recipes: line"
```

Comment:
Style/ExtraSpacing: Unnecessary spacing detected.<br>Style/SingleSpaceBeforeFirstArg: Put one space between the method name and the first argument.